sale_stock_operating_unit/readme/CONFIGURATION.rst

```diff
@@ -0,0 +1,2 @@
+Go to 'Settings / Technical / Actions / User-defined Defaults' and remove
+the default set for the Shop.
```

Comment:
@AaronHForgeFlow is this still necessary?

Reply:
Not sure, I've never use that feature. I have to check

Reply:
I checked and I think that is not necessary